_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -390,28 +325,24 @@ This [HTML document][] has a [keyboard shortcut][] using only a [printable chara
 </html>
 ```
 
-#### Failed Example 6
+### Failed
+
+#### Failed Example 1
 
-This [HTML document][] has a [keyboard shortcut][] using only a [printable character][] which can be remapped, but the remapping [instrument][] is not [visible][].
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and:
+ - doesn't exist any [instrument][] to prevent any result of the [keyboard event][]; nor
+ - the result of the [keyboard event][] is prevented when the [event target][] doesn't have [focus][]; nor
+ - exists any any [instrument][] to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`.
```

Comment:
```suggestion
 - there doesn't exists any [instrument][] to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`.
```